Head-of-Product.md

```diff
@@ -0,0 +1,62 @@
+## About [Snyk](https://snyk.io)
+
+<img src="https://snyk.io/images/snyk-dog.png" width="100" alt="Snyk logo" />
+
+Snyk's mission is to help developers use open source code and stay secure. 
+Open Source code, pulled from npm, Maven, RubyGems and others, is often the majority of code in an application, and yet it receives no security attention. 
+Most developers do nothing to track known vulnerabilities in the packages they use, and trust this code as though its their own. 
+This risk is massive today, and is only made worse by growing use of packages and trends such as Serverless and PaaS, who secure the server's dependencies but leave the app to fend with its dependencies security alone.
```

Comment:
Suggest to rephrase so the note on Serverless and PaaS is more digestible to someone who's not entrenched in the industry:
"This risk is massive today, and is only made worse by the growing use of packages and the simplicity offered by package managers. This is compounded by the fact that unpatched servers, which are the target of many hackers, are slowly being abstracted away by PaaS and Serverless offerings, making vulnerabilities in open source code take center stage"